spiceaidocs/content/en/reference/pod/_index.md

```diff
@@ -178,22 +178,22 @@ dataspaces:
     name: btcusd
 ```
 
-### `dataspaces[*].fields`
+### `dataspaces[*].measurements`
 
-**Required**. The fields to create in the dataspace. If these fields are not populated by either a data connector or through the API, they will use the default value of `0.0`. Each field must be a `float`.
+The measurements to create in the dataspace. If these measurements are not populated by either a data connector or through the API, they will use the default value of `0.0`. Each field must be a `float`.
```

Comment:
This is not 100% accurate right? They will be initialized to a default value of 0.0 then filled or not?

Reply:
That sentence was confusing so I removed it. I wanted to highlight that unless data is provided, it will default to zero.